### PR TITLE
Fix OOD_NAVBAR_TYPE=light not working

### PR DESCRIPTION
**Pull request by robinkar**
Fixed a small bug that caused setting environmental variable OOD_NAVBAR_TYPE to light to not work as expected.

### CODE REVIEW

**Comment by johrstrom** [comment]
Hi! Thanks for the PR.  It looks like this should work, but doesn't?

I had to write test cases to be sure (these currently fail) - can you add these to `test/config/configuration_singleton.rb`?

```ruby
  test "inverse navbar is dark" do
    with_modified_env(OOD_NAVBAR_TYPE: 'inverse') do
      assert_equal 'dark', ConfigurationSingleton.new.navbar_type
    end
  end

  test "dark navbar is dark" do
    with_modified_env(OOD_NAVBAR_TYPE: 'dark') do
      assert_equal 'dark', ConfigurationSingleton.new.navbar_type
    end
  end

  test "default navbar is dark" do
    with_modified_env(OOD_NAVBAR_TYPE: 'default') do
      assert_equal 'light', ConfigurationSingleton.new.navbar_type
    end
  end

  test "light navbar is light" do
    with_modified_env(OOD_NAVBAR_TYPE: 'light') do
      assert_equal 'light', ConfigurationSingleton.new.navbar_type
    end
  end
```

**Comment by robinkar** [comment]
Added the tests, I assume "default navbar is dark" should be "default navbar is light" as in the assert_equal and code.

File: apps/dashboard/test/config/configuration_singleton_test.rb [hunk 1]
```diff
@@ -391,4 +391,28 @@ def config_fixtures
...
+    with_modified_env(OOD_NAVBAR_TYPE: 'dark') do
+      assert_equal 'dark', ConfigurationSingleton.new.navbar_type
+    end
+  end
+
+  test "default navbar is light" do
+    with_modified_env(OOD_NAVBAR_TYPE: 'default') do
+      assert_equal 'light', ConfigurationSingleton.new.navbar_type
+    end
+  end
```
Comment (johrstrom) [inline apps/dashboard/test/config/configuration_singleton_test.rb:411]: I see, that is misleading. Can I ask for 1 more test `no navbar environment variable is dark`?

```ruby
  test "no navbar environment variable is dark" do
    assert_equal 'dark', ConfigurationSingleton.new.navbar_type
  end
```

**Review by johrstrom — APPROVED** [review]
Thanks for the fix!

**Pull request merged**